### ISSUE TITLE
Duplicate rows on initial replication

### ISSUE THREAD

**Issue by mullr** [2ndQuadrant/pglogical#59]
postgresql version: 9.4.9
pglogical verison: 1.2.1

We are seeing intermittent failures in initial replication that appear to be caused by some rows getting replicated twice. We're still working on a repro for this, but it's something along these lines:

1. Have a table on the source-side with a uuid primary key (not sure if that matters), some other columns, and a last_active timestamp (which is populated by the application, not the database)
2. Add all tables in the public schema to the default replication set, using replication_set_add_all_tables with synchronize_data=true
3. While the source continues to serve requests as normal, create a subscription on the replica
  a. All the tables get created on the replica side
  b. Two rows appear on the replica side, each with the same uuid primary key, but with different last_active timestamps
  c. PostgreSQL logs an error, saying "could not create unique index ..." for the primary key
  d. pglogical logs an error saying "subscriber <name> initialization failed during nonrecoverable step (c), please try the setup again"

It /appears/ to be triggered to be correlated with write activity on the source side, but that's only anecdotal at this point.

**Comment by holguinj** [comment]
I've been trying to reproduce this to no avail so far. Here's what I did:

1. create two nodes in two instances of postgres 9.4.10 (primary + replica), without a subscription
2. create a table on the primary with 10,000 rows and run a script to constantly update them (leaving the primary key alone)
3. while that updating is going on, create a subscription from the replica to the primary

The result was a well-replicated database without any errors. I've made [this repo](https://github.com/holguinj/pglogical-duplicate-row-bug) to help anyone else who wants to try to reproduce this, if they're so inclined.
